client/src/App.tsx

```diff
@@ -23,6 +24,7 @@ function App(): JSX.Element {
                 <Route exact path="/dashboard">
                   <Dashboard />
                 </Route>
+                <Route exact path="/mysitter" component={Bookings} />
```

Comment:
Please use hyphenated case for route names.

Eg: `edit-profile`

Reply:
done